fastlane/Fastfile

```diff
@@ -54,10 +55,12 @@ before_all do |lane|
 
   # Check that the env files exist
   unless is_ci || File.file?(USER_ENV_FILE_PATH)
-    UI.user_error!("~/.wpios-env.default not found: Please copy fastlane/env/user.env-example to #{USER_ENV_FILE_PATH} and fill in the values")
+    example_path = 'fastlane/env/user.env-example '
+    msg = "#{ENV_FILE_NAME} not found: Please copy #{example_path} to #{USER_ENV_FILE_PATH} and fill in the values"
+    UI.user_error!(msg)
```

Comment:
super-nitpicky-me: not sure it's worth having `msg = …` compared to directly calling `UI.user_error!` with the string literal 😛

Reply:
I think I originally split it to address a line length violation. But now that I checked, I don't get that violation anymore...

Addressed in 2b76a1a04b.